proto/invocation.proto

```diff
@@ -532,3 +541,75 @@ message GetTrendResponse {
 
   bool has_invocation_stat_percentiles = 4;
 }
+
+enum Metric {
+  UNKNOWN_METRIC = 0;
+  DURATION_USEC_METRIC = 1;
+  CAS_CACHE_MISSES_METRIC = 2;
+  EXECUTION_QUEUE_TIME_USEC_METRIC = 3;
+}
+
+message StatFilter {
+  Metric metric = 1;
+  int64 min = 2;
+  int64 max = 3;
+}
+
+// Fetches a heatmap for the requested metric--a heatmap is basically a set of
+// columns, where each column represents a histogram for that metric in a window
+// of time.
+message GetStatHeatmapRequest {
+  context.RequestContext request_context = 1;
+  TrendQuery query = 2;
+  Metric metric = 3;
```

Comment:
MetricType to be clearer. My first reaction without looking up is that it's a message that include different metrics.

Reply:
Sure yeah, done.